### ISSUE TITLE
Install instructions do not work

### ISSUE THREAD

**Issue by birdayz** [stephenafamo/golangci-server#2]
```
$ go install github.com/stephenafamo/golangci-server@latest
go: github.com/stephenafamo/golangci-server@latest (in github.com/stephenafamo/golangci-server@v0.0.0-20220804181834-3e77e3ca11cf):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

**Comment by stephenafamo** [comment]
I forgot about this `go install` limitation. I'll fix this soon.

**Comment by stephenafamo** [comment]
Try again. Fixed in commit [f75aef8](https://github.com/stephenafamo/golangci-server/commits/main).

If `@latest` doesn't work because of the cache in `pkg.go.dev`, try `go install github.com/stephenafamo/golangci-server@f75aef8`
